build/quickstarts-showcase/src/main/resources/META-INF/resources/index.html

```diff
@@ -189,6 +189,30 @@ <h5>
                     </div>
                 </div>
             </div>
+            <div class="col mb-4">
+                <div class="card">
+                    <div class="card-header">
+                        <h5>
+                            <i class="fas fa-calendar"></i>
+                            Employee Scheduling
+                            <!-- TODO: Blog Post -->
+                            <a href="https://www.optaplanner.org/blog/2021/10/14/OrderPickingQuickstart.html" target="_blank" title="About this quickstart" class="float-right">
+                                <span class="fas fa-info-circle"></span>
+                            </a>
+                        </h5>
+                    </div>
+                    <img src="screenshot/quarkus-employee-scheduling-screenshot.png" class="card-img-top mt-3" alt="Screenshot"/>
+                    <div class="card-body">
+                        <p class="card-text">
+                            Schedule shifts to employees, accounting for employee availabilities.
```

Comment:
Let's mention skills too. 
```suggestion
                            Schedule shifts to employees, accounting for employee availability and shift skill requirements.
```
Let's use the exact same sentence for the other mentions (README etc).